dotnet-client-libraries/YamlDotNet/Tests/BasicSerializeAndDeserializeTests.cs

```diff
@@ -0,0 +1,30 @@
+using App.Models;
+using App.UseCases;
+
+namespace Tests;
+
+public class BasicSerializeAndDeserializeTests
+{
+    [Fact]
+    public void WhenProductIsSerialized_ThenYamlIsReturned()
+    {
+        // Arrange
+        var product = new Product { Id = 1, Name = "Test", Price = 10.0m };
+        var expectedYaml = "Id: 1\nName: Test\nPrice: 10.0\n";
```

Comment:
On my Windows system this test is failing because the actual value returned is:

```
"Id: 1\r\nName: Test\r\nPrice: 10.0\r\n"
```

Maybe something like this would be better:

```suggestion
        var expectedYaml = $"Id: 1{Environment.NewLine}Name: Test{Environment.NewLine}Price: 10.0{Environment.NewLine}";
```

Reply:
@grantwinney Done